scripts/mkcloud

```diff
@@ -41,6 +40,10 @@ fi
 # causes of this is violation of the common shell coding standard which
 # uses uppercase for environment variables and constants, and lowercase
 # for local variables.
+: ${cinder_backend:=''}
+: ${cinder_netapp_storage_protocol:=iscsi}
+: ${cinder_netapp_login:=openstack}
+: ${cinder_netapp_password:=''}
```

Comment:
> there is a sanity check - when applying the proposal. the parameters are mapped to the attributes and the proposal will fail if something is wrong.

I meant the sanity check in mkcloud, which would hit earlier, so you wouldn't have to wait for it so long to fail

> And another thing - what if we want to test netapp NFS and ISCSI together?

good point, we could do sth like:
``` bash
if [[ $want_cindermultibackend = 1 ]] ; then
    ...
    if [[ $cinder_backend = "netapp" ]]; then
        if [[ $cinder_netapp_storage_protocol = "iscsi" ]] ; then
            cinder_netapp_storage_protocol = "nfs"
        else
            cinder_netapp_storage_protocol = "iscsi"
        fi
        cinder_netapp_proposal_configuration
    fi
fi
```

I will have to adapt `cinder_netapp_proposal_configuration` as well to take the protocol into the `backend_name` that we can test both.
As discussed in with @jdsn I will open another PR for that after this one is merged